helm/parseable/Chart.yaml

```diff
@@ -3,4 +3,4 @@ name: parseable
 description: Helm chart for Parseable Server
 type: application
 version: 0.0.1
-AppVersion: ""
+AppVersion: "sha-b6d9366"
```

Comment:
This should be a proper release version, we can update once we make the `0.0.1` release?

Reply:
yes once release is cut, we have to bump versions